Tensile/TensileCreateLib/ParseArguments.py

```diff
@@ -297,6 +297,8 @@ def parseArguments(input: List[str] | None = None) -> Dict[str, Any]:
     if args.GenerateSourcesAndExit:
         # Generated sources are preserved and go into output directory
         arguments["WorkingPath"] = arguments["OutputPath"]
+    if args.PrintLevel == 0:
```

Comment:
I do not think it is a good way to ignore warning at any print level.

Reply:
Any request to do so?

Reply:
I might misunderstand this.
Isn't this related to compiler warning?

Reply:
If this is not related to compiler warning, I am OK with this.

Reply:
This is not directly related to compiler warnings, though some TensileCreateLibrary warnings related to compiler issues may be emitted. Please see my newest comment.